docs/spec/api.md

```diff
@@ -1,3 +1,7 @@
+page_title: Docker Registry HTTP API V2
```

Comment:
@stevvooe Actually that wasn't working entirely on Mac.  
```
   grep -vE '^<!--GITHUB|IGNORES-->'
```
This is also more self documenting in terms of comments.